.github/workflows/protected-files.yaml

```diff
@@ -6,6 +6,9 @@ jobs:
   protected-files:
     name: Protected Files
 
+    # Always allow bot account 'azure-sdk' to update protected files
+    if: ${{ github.event.pull_request.user.login != 'azure-sdk' }}
```

Comment:
I wonder if there is a easy way to do something like this to check for people in the azure-sdk-eng group as well.

Reply:
I was thinking of exactly this, and I can add it.  It will require an API call, but it's pretty easy.  Will create a PR.